runtime/queries/ldif/highlights.scm

```diff
@@ -0,0 +1,21 @@
+(comment) @comment
+
+((distinguishedName
+  (name
+    (name_componet
+      (attributeTypeAndValue
+        (attributeType) @comment
+        (string) @type.parameter
+          )))) @comment)
+
+
+(dn_spec) @constant
+(changerecord) @constant
+(mod_spec) @constant
+
+(attributeType) @type.parameter
+(change_modify) @string
+
+(value_spec) @keyword
+
+(ERROR) @error
```

Comment:
```suggestion
```

we want to make error highlighting a config option instead. For now we remove the pattern from most languages since it's usually a very noisy highlight